doc/api/worker_threads.md

```diff
@@ -4,21 +4,20 @@
 
 > Stability: 1 - Experimental
 
-The `worker` module provides a way to create multiple environments running
-on independent threads, and to create message channels between them. It
-can be accessed using the `--experimental-worker` flag and:
+The `worker_threads` module enables the use of threads with message channels
+between them. To access it, use the `--experimental-worker` flag and:
```

Comment:
https://github.com/nodejs/node/commit/63d4cae009e114127e1a80644e4bffc019e2f4a7 had landed, flag is gone.

Reply:
Thanks. I'll update!